content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,146 @@ enum Level {
+   Add [`directUrl`](/reference/api-reference/prisma-schema-reference#fields) property in `datasource` block in the `schema.prisma` file.
 
-   ```diff file=.env
-   - DATABASE_URL = "mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority"
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   ```diff file=prisma/schema.prisma
+   datasource db {
+    provider = "postgresql"
+    url = env("DATABASE_URL")
+    +directUrl = env("DIRECT_URL")
+   }
    ```
 
+> The `directUrl` field is not required for Prisma Accelerate. It allows you to easily perform database migrations and introspections.
+
 2. In your code editor, add the connection string in the `wrangler.toml` file.
 
    ```diff file=wrangler.toml
-     name = "prisma-mongodb-cloudflare"
+     name = "prisma-cloudflare-accelerate"
      main = "src/main.ts"
      compatibility_date = "2022-11-07"
 
    + [vars]
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   + DATABASE_URL = "prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
+   ```
+
+3. Install the Accelerate Client extension
+
+   ```bash
+   npm install @prisma/extension-accelerate
    ```
 
 You are now ready to generate a Prisma Client.
 
-## 8. Generate a Prisma Client
+## 6. Migrate your schema and generate a Prisma Client
 
-Next, you'll generate a Prisma Client that connects through the [Data Proxy](/data-platform/classic-projects/data-proxy) over HTTP.
+To apply migrations execute the following command:
+
+```
+prisma migrate deploy
+```
+
+Next, you'll generate Prisma Client that connects to your database through [Prisma Accelerate](/data-platform/accelerate) over HTTP.
 
 ```terminal
 npx prisma generate --no-engine
 ```
 
 <Admonition>
 
-Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--data-proxy` flag.
+Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--accelerate` flag.
 
 ```terminal
-npx prisma generate --data-proxy
+npx prisma generate --accelerate
 ```
 
 </Admonition>
 
 The generated Client has a smaller bundle size and is optimized for edge environments like Cloudflare Workers.
 
-The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by the Data Proxy.
+The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by Prisma Accelerate.
 
-## 9. Develop the Cloudflare Worker function
+## 7. Develop the Cloudflare Worker function
 
 You're now ready to create a Cloudflare Worker. Create a `src/index.ts` file with the following code:
 
 ```ts
 import { PrismaClient } from '@prisma/client/edge'
```

Comment:
Does this work? Do we have an example app that validates this works in wrangler v3?